src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -525,6 +535,60 @@ class PickPage extends Component {
       .catch(() => { this.props.hideSpinner(); });
   }
 
+  // Returns indexes of rows with quantity picked 0, and without subitems.
+  getRowsWithEmptyPicks(pickPageItems) {
+    return pickPageItems.reduce((acc, item, index) => {
+      // When the quantity picked is equal to 0 we have to check its subitems,
+      // because the item can be edited to 0, not cleared.
+      if (!item.quantityPicked && !item.picklistItems.length) {
+        return [...acc, index + 1];
```

Comment:
```suggestion
  getIndexesOfRowsWithEmptyPicks(pickPageItems) {
    return pickPageItems.reduce((acc, item, index) => {
      // When the quantity picked is equal to 0 we have to check its subitems,
      // because the item can be edited to 0, not cleared.
      if (!item.quantityPicked && !item.picklistItems.length) {
        return [...acc, index];
```

I saw Arturs comment suggesting adding a comment which you did, but I feel strongly that this should have a function name with clear intent, this should say that it returns indexes (normal indexes, 0,1,2,3...) and when we need to display these indexes to a user just do `indexes.map(it => it+1)`

I saw you use this same function later where you had to do this `indexes.includes(index + 1)` which doesn't sit right with me :/ 
Also the name might be confusing since `getRowsWithEmptyPicks` suggests that we return `rows` (object of values)